src/actions.js

```diff
@@ -196,6 +198,15 @@ async function deploy(args) {
   logger.info('targeting services', services);
   const descriptions = await ecs.describeServices({ cluster, services: serviceNames }).promise();
 
+  if (!force) {
+    // prompt user for deploy
+    const answer = await prompt('Are you sure you want to continue with the deploy? [Y/n]').run();
```

Comment:
would be awesome if it printed what it parsed from the command line so that the user could compare what they asked for with what they are getting

Reply:
I think it does, it prompts after the output. For example:
```
➤  node ./lib/pecs.js release -c dev-cluster -s -t master
info:    requested release cluster=dev-cluster, services=[], tag=master
info:    targeting services 0=parse-server, 1=aggregation, 2=admin, 3=job-queue-proxy, 4=salesforce, 5=diff-man, 6=parse-proxy, 7=csv-import, 8=cron, 9=worker
? Are you sure you want to continue with the deploy? [Y/n]
```